watertap/unit_models/tests/test_crystallizer.py

```diff
@@ -286,6 +290,19 @@ def test_calculate_scaling(self, Crystallizer_frame):
 
     @pytest.mark.component
     def test_initialize(self, Crystallizer_frame):
+        # Add costing function, then initialize
+        m = Crystallizer_frame
+        m.fs.costing = WaterTAPCosting()
+        m.fs.unit.costing = UnitModelCostingBlock(
+            default={
+                "flowsheet_costing_block": m.fs.costing,
+                "costing_method_arguments": {
+                    "cost_type": CrystallizerCostType.mass_basis
+                },
+            },
+        )
+        m.fs.costing.cost_process()
+
```

Comment:
Are these tests in the right place?

On one hand, our general philosophy has been to put tests in the same directory as the code, i.e., in the `watertap/costing/tests/` directory. On the other hand, breaking them out into the unit model tests makes a huge amount of sense as well -- we already have the code to set up the unit model, etc.

Reply:
I could do this. However, are we sure that is how we want to handle the costing tests? That just seems to add a lot of unnecessary overhead to me.

Reply:
I'm happy with the tests as they are because the costing tests are *not* currently centralized in `watertap/costing/tests/`.

Looking at the code I'm not sure centralizing *is* the right idea -- perhaps it should be with the unit test as these are.